go.mod

```diff
@@ -1,12 +1,20 @@
-module github.com/asdine/storm
+module github.com/asdine/storm/v2
 
 require (
+	github.com/DataDog/zstd v1.3.5 // indirect
 	github.com/Sereal/Sereal v0.0.0-20181211220259-509a78ddbda3
+	github.com/asdine/storm v2.1.2+incompatible
```

Comment:
I don't think storm should be depending on an older version of itself after this change.  Did you miss an import change perhaps?

Reply:
Please merge or reference https://github.com/Xe/storm/pull/1 for the complete fix.

Reply:
thank you @chappjc